bundles/org.openhab.binding.miio/src/test/java/org/openhab/binding/miio/internal/ReadmeHelper.java

```diff
@@ -65,14 +67,11 @@ public static void main(String[] args) {
         StringWriter itemFileExamples = rm.itemFileExamples();
         LOGGER.info("## Done");
         try {
-            File file = new File(BASEFILE);
-            String baseDoc = FileUtils.readFileToString(file, "UTF-8");
+            String baseDoc = new String(Files.readAllBytes(Paths.get(BASEFILE)));
             String nw = baseDoc.replaceAll("!!!devices", deviceList.toString())
                     .replaceAll("!!!channelList", channelList.toString())
                     .replaceAll("!!!itemFileExamples", itemFileExamples.toString());
-
-            File newDocfile = new File("README.md");
-            FileUtils.writeStringToFile(newDocfile, nw, "UTF-8");
+            Files.write(Paths.get(OUTPUTFILE), nw.getBytes());
```

Comment:
The default charset varies from system to system so it is better to specify it yourself instead of using the default.